### PR TITLE
April 2018 items Update

### PR DESCRIPTION
**Pull request by Ninjistix**
new and improved version of #4679 
- Contains Jan/Feb/Mar changes

various other adjustments are listed in the commit comments.

new client version is 30180329_1 (my bad had pol ver before)

### CODE REVIEW

**Comment by TeoTwawki** [comment]
contains same things we want changes on from the other pr from the quick glance I gave it.

**Comment by Ninjistix** [comment]
this doesn't contain any ah category adjustments that were in the previous PR, the only ah category changes I made were to the default category from 99 back to 0 as you stated it should be, everything else is from the version changes.

**Comment by zynjec** [comment]
Has the base delay of H2H been changed? Everything I'm finding is saying 480 base which would put those delay numbers incorrect for all your changes.

eg: Lizard Cesti you changed delay from 528 to 531, but should be 480 (base) + 48 (weapon) for a total of 528.

**Comment by Ninjistix** [comment]
@zynjec maybe it added wrong will double check when I get back in the office.

**Comment by zynjec** [comment]
```
dish_of_spaghetti_nero_di_seppia_+1.lua → ish_of_spag_nero_di_seppia_+1.lua
serving_of_patriarch_sautee.lua → erving_of_patriarch_saute.lua
```
Typos in rename.

File: sql/item_weapon.sql [hunk 1]
```diff
@@ -3709,7 +3714,7 @@ INSERT INTO `item_weapon` VALUES (20756,'soulcleaver',4,0,228,228,188,2,1,489,28
 INSERT INTO `item_weapon` VALUES (20757,'foreshock_sword',4,0,242,242,188,2,1,504,273,0);
 INSERT INTO `item_weapon` VALUES (20758,'mekosuchus_blade',4,0,228,228,188,2,1,504,273,0);
 INSERT INTO `item_weapon` VALUES (20759,'macbain',4,0,242,242,188,2,1,510,268,0);
-INSERT INTO `item_weapon` VALUES (20760,'home._claymore',4,0,215,215,167,2,1,444,228,0);
+INSERT INTO `item_weapon` VALUES (20760,'home._claymore',4,0,215,215,167,2,1,444,240,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 2]
```diff
@@ -3806,7 +3811,7 @@ INSERT INTO `item_weapon` VALUES (20855,'devivifier',6,0,242,242,188,2,1,504,285
 INSERT INTO `item_weapon` VALUES (20856,'phreatic_axe',6,0,242,242,188,2,1,504,274,0);
 INSERT INTO `item_weapon` VALUES (20857,'svarga',6,0,242,242,188,2,1,545,286,0);
 INSERT INTO `item_weapon` VALUES (20858,'lightreaver',6,0,242,242,188,2,1,504,273,0);
-INSERT INTO `item_weapon` VALUES (20859,'home._labrys',6,0,215,215,167,2,1,504,259,0);
+INSERT INTO `item_weapon` VALUES (20859,'home._labrys',6,0,215,215,167,2,1,504,273,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 3]
```diff
@@ -3846,7 +3851,7 @@ INSERT INTO `item_weapon` VALUES (20898,'triska_scythe',7,0,242,242,188,2,1,528,
 INSERT INTO `item_weapon` VALUES (20899,'triska_scythe_+1',7,0,242,242,188,2,1,513,287,0);
 INSERT INTO `item_weapon` VALUES (20900,'meekculler',7,0,242,242,188,2,1,528,290,0);
 INSERT INTO `item_weapon` VALUES (20901,'inanna',7,0,242,242,188,2,1,550,288,0);
-INSERT INTO `item_weapon` VALUES (20903,'home._scythe',7,0,215,215,167,2,1,528,271,0);
+INSERT INTO `item_weapon` VALUES (20903,'home._scythe',7,0,215,215,167,2,1,528,286,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 4]
```diff
@@ -3887,7 +3892,7 @@ INSERT INTO `item_weapon` VALUES (20943,'gae_derg_+1',8,0,242,242,188,1,1,540,29
 INSERT INTO `item_weapon` VALUES (20944,'pelagos_lance',8,0,242,242,188,1,1,507,275,0);
 INSERT INTO `item_weapon` VALUES (20945,'nativus_halberd',8,0,228,228,188,1,1,396,207,0);
 INSERT INTO `item_weapon` VALUES (20946,'olyndicus',8,0,242,242,188,1,1,528,277,0);
-INSERT INTO `item_weapon` VALUES (20947,'homestead_lance',8,0,215,215,167,1,1,492,252,0);
+INSERT INTO `item_weapon` VALUES (20947,'homestead_lance',8,0,215,215,167,1,1,492,266,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 5]
```diff
@@ -3932,7 +3937,7 @@ INSERT INTO `item_weapon` VALUES (20989,'izuna',9,0,242,242,188,2,1,237,124,0);
 INSERT INTO `item_weapon` VALUES (20990,'jushimatsu',9,0,242,242,188,2,1,227,122,0);
 INSERT INTO `item_weapon` VALUES (20991,'chidori',9,0,242,242,188,2,1,227,122,0);
 INSERT INTO `item_weapon` VALUES (20992,'taikogane',9,0,188,188,146,2,1,227,110,0);
-INSERT INTO `item_weapon` VALUES (20993,'senkutanto',9,0,203,203,191,2,1,185,95,0);
+INSERT INTO `item_weapon` VALUES (20993,'senkutanto',9,0,203,203,191,2,1,185,100,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 228
ilvl_parry 228
ilvl_macc 215

File: sql/item_weapon.sql [hunk 6]
```diff
@@ -3974,7 +3979,7 @@ INSERT INTO `item_weapon` VALUES (21034,'kunimune',10,0,242,242,188,2,1,450,243,
 INSERT INTO `item_weapon` VALUES (21035,'kunimune_+1',10,0,242,242,188,2,1,437,244,0);
 INSERT INTO `item_weapon` VALUES (21036,'atakigiri',10,0,242,242,188,2,1,480,258,0);
 INSERT INTO `item_weapon` VALUES (21037,'nenekirimaru',10,0,242,242,188,2,1,484,254,0);
-INSERT INTO `item_weapon` VALUES (21038,'senkuto',10,0,203,203,167,2,1,420,215,0);
+INSERT INTO `item_weapon` VALUES (21038,'senkuto',10,0,203,203,167,2,1,420,227,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 228
ilvl_parry 228
ilvl_macc 118

File: sql/item_weapon.sql [hunk 7]
```diff
@@ -4042,7 +4047,7 @@ INSERT INTO `item_weapon` VALUES (21104,'eosuchus_club',11,0,242,242,215,3,1,264
 INSERT INTO `item_weapon` VALUES (21105,'nehushtan',11,0,228,228,188,3,1,352,184,0);
 INSERT INTO `item_weapon` VALUES (21107,'kyuka_uchiwa',11,0,0,0,0,3,1,216,1,0);
 INSERT INTO `item_weapon` VALUES (21108,'kyuka_uchiwa_+1',11,0,0,0,0,3,1,210,2,0);
-INSERT INTO `item_weapon` VALUES (21109,'homestead_wand',11,0,215,215,191,3,1,216,111,0);
+INSERT INTO `item_weapon` VALUES (21109,'homestead_wand',11,0,215,215,191,3,1,216,117,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 215

File: sql/item_weapon.sql [hunk 8]
```diff
@@ -4103,8 +4108,8 @@ INSERT INTO `item_weapon` VALUES (21167,'frazil_staff',12,0,242,242,188,3,1,366,
 INSERT INTO `item_weapon` VALUES (21168,'ungeri_staff',12,0,242,242,188,3,1,412,220,0);
 INSERT INTO `item_weapon` VALUES (21169,'keraunos',12,0,242,242,228,3,1,435,228,0);
 INSERT INTO `item_weapon` VALUES (21170,'nilgal_pole',12,0,242,242,188,3,1,412,223,0);
-INSERT INTO `item_weapon` VALUES (21171,'homestead_staff',12,0,215,215,203,3,1,366,188,0);
-INSERT INTO `item_weapon` VALUES (21172,'home._scepter',12,0,215,215,167,3,1,402,206,0);
+INSERT INTO `item_weapon` VALUES (21171,'homestead_staff',12,0,215,215,203,3,1,366,198,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 282

File: sql/item_weapon.sql [hunk 9]
```diff
@@ -4103,8 +4108,8 @@ INSERT INTO `item_weapon` VALUES (21167,'frazil_staff',12,0,242,242,188,3,1,366,
...
+INSERT INTO `item_weapon` VALUES (21172,'home._scepter',12,0,215,215,167,3,1,402,217,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 10]
```diff
@@ -4156,7 +4161,7 @@ INSERT INTO `item_weapon` VALUES (21221,'brahmastra',25,0,242,0,0,1,1,600,261,0)
 INSERT INTO `item_weapon` VALUES (21222,'mengado',25,0,228,0,0,1,1,300,147,0);
 INSERT INTO `item_weapon` VALUES (21223,'mengado_+1',25,0,228,0,0,1,1,291,148,0);
 INSERT INTO `item_weapon` VALUES (21224,'phaosphaelia',25,0,242,0,0,1,1,735,279,0);
-INSERT INTO `item_weapon` VALUES (21226,'homestead_bow',25,0,215,0,0,1,1,540,226,0);
+INSERT INTO `item_weapon` VALUES (21226,'homestead_bow',25,0,215,0,0,1,1,540,238,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242

File: sql/item_weapon.sql [hunk 11]
```diff
@@ -4178,7 +4183,7 @@ INSERT INTO `item_weapon` VALUES (21244,'ahkormaar_bow_+1',25,0,0,0,0,1,1,524,10
 INSERT INTO `item_weapon` VALUES (21245,'guglielmos_bow',26,0,0,0,0,1,1,216,30,0);
 INSERT INTO `item_weapon` VALUES (21246,'gastraphetes',26,0,242,0,0,1,1,432,93,0);
 INSERT INTO `item_weapon` VALUES (21247,'gastraphetes',26,0,242,0,0,1,1,432,93,0);
-INSERT INTO `item_weapon` VALUES (21248,'home._bowgun',26,0,215,0,0,1,1,432,107,0);
+INSERT INTO `item_weapon` VALUES (21248,'home._bowgun',26,0,215,0,0,1,1,432,135,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242

File: sql/item_weapon.sql [hunk 12]
```diff
@@ -4202,7 +4207,7 @@ INSERT INTO `item_weapon` VALUES (21268,'death_penalty',26,1,269,0,0,1,1,480,101
 INSERT INTO `item_weapon` VALUES (21269,'armageddon',26,1,269,0,0,1,1,582,143,0);
 INSERT INTO `item_weapon` VALUES (21273,'nibiru_gun',26,1,242,0,0,1,1,600,119,0);
 INSERT INTO `item_weapon` VALUES (21274,'donar_gun',26,1,228,0,0,1,1,548,103,0);
-INSERT INTO `item_weapon` VALUES (21275,'homestead_gun',26,1,215,0,0,1,1,600,128,0);
+INSERT INTO `item_weapon` VALUES (21275,'homestead_gun',26,1,215,0,0,1,1,600,130,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242

File: sql/item_weapon.sql [hunk 13]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
 INSERT INTO `item_weapon` VALUES (21497,'rapid_broth',0,0,0,0,0,0,1,2216,5203,0);
 INSERT INTO `item_weapon` VALUES (21498,'crackling_broth',0,0,0,0,0,0,1,2217,3412,0);
 INSERT INTO `item_weapon` VALUES (21499,'creepy_broth',0,0,0,0,0,0,1,2218,3669,0);
-INSERT INTO `item_weapon` VALUES (21504,'arasy_sainti',1,0,242,242,188,0,1,531,96,0);
-INSERT INTO `item_weapon` VALUES (21505,'arasy_sainti_+1',1,0,242,242,188,0,1,529,97,0);
-INSERT INTO `item_weapon` VALUES (21506,'jolt_counter',1,0,242,242,188,0,1,576,148,0);
-INSERT INTO `item_weapon` VALUES (21507,'jolt_counter_+1',1,0,242,242,188,0,1,571,149,0);
+INSERT INTO `item_weapon` VALUES (21504,'arasy_sainti',1,0,242,242,188,0,1,561,110,0);
+INSERT INTO `item_weapon` VALUES (21505,'arasy_sainti_+1',1,0,242,242,188,0,1,559,111,0);
+INSERT INTO `item_weapon` VALUES (21506,'jolt_counter',1,0,242,242,188,0,1,566,183,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: delay 606

File: sql/item_weapon.sql [hunk 14]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21507,'jolt_counter_+1',1,0,242,242,188,0,1,564,184,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: delay 601

File: sql/item_weapon.sql [hunk 15]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21511,'hep._baghnakhs',1,0,0,0,0,0,1,570,161,0);
+INSERT INTO `item_weapon` VALUES (21512,'hep._baghnakhs_+1',1,0,0,0,0,0,1,561,162,0);
+INSERT INTO `item_weapon` VALUES (21513,'raetic_baghnakhs',1,0,0,0,0,0,1,531,163,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: delay 570
ilvl_skill 242
ilvl_macc 215

if ilvl_parry is used for guard, ilvl_parry also needs to be 242, if not, ilvl_guard needs to be created, and ilvl_guard 242.

File: sql/item_weapon.sql [hunk 16]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21514,'rae._baghnakhs_+1',1,0,0,0,0,0,1,546,164,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: delay 561
ilvl_skill 242
ilvl_macc 215

if ilvl_parry is used for guard, ilvl_parry also needs to be 242, if not, ilvl_guard needs to be created, and ilvl_guard 242.

File: sql/item_weapon.sql [hunk 17]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
 INSERT INTO `item_weapon` VALUES (21554,'arasy_knife',2,0,242,242,188,0,1,183,94,0);
 INSERT INTO `item_weapon` VALUES (21555,'arasy_knife_+1',2,0,242,242,188,0,1,178,95,0);
+INSERT INTO `item_weapon` VALUES (21556,'beryllium_kris',2,0,0,0,0,0,1,192,125,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 228
ilvl_parry 228
ilvl_macc 188

File: sql/item_weapon.sql [hunk 18]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21557,'beryllium_kris_+1',2,0,0,0,0,0,1,187,126,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 228
ilvl_parry 228
ilvl_macc 188

File: sql/item_weapon.sql [hunk 19]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21558,'rune_kris',2,0,0,0,0,0,1,192,24,0);
+INSERT INTO `item_weapon` VALUES (21559,'raetic_kris',2,0,0,0,0,0,1,192,128,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 215

File: sql/item_weapon.sql [hunk 20]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21560,'raetic_kris_+1',2,0,0,0,0,0,1,187,129,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 215

File: sql/item_weapon.sql [hunk 21]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
 INSERT INTO `item_weapon` VALUES (21604,'arasy_sword',3,0,242,242,188,0,1,240,123,0);
 INSERT INTO `item_weapon` VALUES (21605,'arasy_sword_+1',3,0,242,242,188,0,1,236,124,0);
 INSERT INTO `item_weapon` VALUES (21606,'enriching_sword',3,0,242,242,188,0,1,240,156,0);
 INSERT INTO `item_weapon` VALUES (21607,'enr._sword_+1',3,0,242,242,188,0,1,233,157,0);
+INSERT INTO `item_weapon` VALUES (21608,'onion_sword_ii',3,0,0,0,0,0,1,240,1,0);
+INSERT INTO `item_weapon` VALUES (21609,'save_the_queen_ii',3,0,0,0,0,0,1,264,1,0);
+INSERT INTO `item_weapon` VALUES (21610,'hepatizon_rapier',3,0,0,0,0,0,1,224,146,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 201

File: sql/item_weapon.sql [hunk 22]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21611,'hep._rapier_+1',3,0,0,0,0,0,1,218,147,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 201

File: sql/item_weapon.sql [hunk 23]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21612,'raetic_blade',3,0,0,0,0,0,1,236,157,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 215

File: sql/item_weapon.sql [hunk 24]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21613,'raetic_blade_+1',3,0,0,0,0,0,1,230,158,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 215

File: sql/item_weapon.sql [hunk 25]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21614,'hepatizon_sapara',3,0,0,0,0,0,1,236,154,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 26]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21615,'hep._sapara_+1',3,0,0,0,0,0,1,230,155,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 27]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
 INSERT INTO `item_weapon` VALUES (21654,'arasy_claymore',4,0,242,242,188,0,1,489,251,0);
 INSERT INTO `item_weapon` VALUES (21655,'arasy_claymore_+1',4,0,242,242,188,0,1,475,252,0);
 INSERT INTO `item_weapon` VALUES (21656,'dyrnwyn',4,0,228,228,228,0,1,480,313,0);
 INSERT INTO `item_weapon` VALUES (21657,'dyrnwyn_+1',4,0,228,228,228,0,1,466,314,0);
+INSERT INTO `item_weapon` VALUES (21659,'beryllium_sword',4,0,0,0,0,0,1,444,289,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 28]
```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
...
+INSERT INTO `item_weapon` VALUES (21660,'bery._sword_+1',4,0,0,0,0,0,1,431,290,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

File: sql/item_weapon.sql [hunk 29]
```diff
@@ -4432,13 +4477,22 @@ INSERT INTO `item_weapon` VALUES (21809,'liberator',7,0,269,269,242,2,1,528,330,
 INSERT INTO `item_weapon` VALUES (21810,'redemption',7,0,269,269,242,2,1,502,354,0);
 INSERT INTO `item_weapon` VALUES (21812,'arasy_scythe',7,0,242,242,188,2,1,528,271,0);
 INSERT INTO `item_weapon` VALUES (21813,'arasy_scythe_+1',7,0,242,242,188,2,1,513,272,0);
+INSERT INTO `item_weapon` VALUES (21815,'maliya_sickle',7,0,0,0,0,0,1,501,327,0);
+INSERT INTO `item_weapon` VALUES (21816,'maliya_sickle_+1',7,0,0,0,0,0,1,490,328,0);
```
Comment (zynjec) [inline sql/item_weapon.sql]: ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

**Comment by zynjec** [comment]
We'll work under the assumption item flags are correct at the moment, there are 100 EX items on the AH still.

```
select itemid,name,aH from item_basic where flags & 0x4000 and aH != 0;
```
item_basic
---
| itemid | name | aH | 
| ---: | --- | ---: | 
| 1 | pile_of_chocobo_bedding | 34 | 
| 7 | gold_bed | 34 | 
| 28 | drawing_desk | 34 | 
| 105 | sky_pot | 34 | 
| 106 | red_jar | 34 | 
| 129 | imperial_standard | 34 | 
| 131 | stationery_set | 34 | 
| 134 | copy_of_emeralda | 34 | 
| 140 | dream_platter | 34 | 
| 141 | dream_coffer | 34 | 
| 142 | drogarogas_fang | 34 | 
| 181 | san_dorian_flag | 34 | 
| 182 | bastokan_flag | 34 | 
| 183 | windurstian_flag | 34 | 
| 184 | jeunoan_flag | 34 | 
| 190 | copy_of_lines_and_space | 34 | 
| 191 | fishing_hole_map | 34 | 
| 192 | copy_of_hoary_spire | 34 | 
| 193 | adventuring_certificate | 34 | 
| 196 | melodious_egg | 34 | 
| 197 | clockwork_egg | 34 | 
| 198 | gilt_tapestry | 34 | 
| 199 | hatchling_egg | 34 | 
| 202 | golden_fleece | 36 | 
| 203 | bomb_lantern | 34 | 
| 204 | pumpkin_lantern | 34 | 
| 205 | mandragora_lantern | 34 | 
| 246 | blackhorn | 34 | 
| 247 | stripehorn | 34 | 
| 248 | whitehorn | 34 | 
| 256 | hume_♂_mannequin | 34 | 
| 257 | hume_♀_mannequin | 34 | 
| 258 | elvaan_♂_mannequin | 34 | 
| 259 | elvaan_♀_mannequin | 34 | 
| 260 | tarutaru_♂_mannequin | 34 | 
| 261 | tarutaru_♀_mannequin | 34 | 
| 262 | mithra_mannequin | 34 | 
| 263 | galka_mannequin | 34 | 
| 264 | stuffed_chocobo | 34 | 
| 265 | adamantoise_statue | 34 | 
| 266 | behemoth_statue | 34 | 
| 267 | fafnir_statue | 34 | 
| 268 | nomad_moogle_statue | 34 | 
| 269 | shadow_lord_statue | 34 | 
| 270 | odin_statue | 34 | 
| 271 | alexander_statue | 34 | 
| 277 | prishe_statue | 34 | 
| 278 | cardian_statue | 34 | 
| 279 | shadow_lord_statue_ii | 34 | 
| 280 | shadow_lord_statue_iii | 34 | 
| 281 | atomos_statue | 34 | 
| 282 | yovra_replica | 34 | 
| 284 | goobbue_statue | 34 | 
| 286 | nanaa_mihgo_statue | 34 | 
| 287 | nanaa_mihgo_statue_ii | 34 | 
| 303 | crystal_stakes_gold_cup | 34 | 
| 304 | crystal_stakes_mythril_cup | 34 | 
| 305 | crystal_stakes_bronze_cup | 34 | 
| 307 | mythril_trophy | 34 | 
| 308 | bronze_trophy | 34 | 
| 309 | yellow_vcs_plaque | 34 | 
| 310 | black_vcs_plaque | 34 | 
| 311 | red_vcs_plaque | 34 | 
| 312 | blue_vcs_plaque | 34 | 
| 313 | green_vcs_plaque | 34 | 
| 320 | harpsichord | 34 | 
| 323 | sprig_of_red_bamboo_grass | 34 | 
| 324 | sprig_of_blue_bamboo_grass | 34 | 
| 325 | sprig_of_green_bamboo_grass | 34 | 
| 334 | blacksmiths_signboard | 34 | 
| 335 | goldsmiths_signboard | 34 | 
| 336 | boneworkers_signboard | 34 | 
| 337 | weavers_signboard | 34 | 
| 338 | culinarians_signboard | 34 | 
| 339 | tanners_signboard | 34 | 
| 340 | fishermans_signboard | 34 | 
| 341 | carpenters_signboard | 34 | 
| 342 | alchemists_signboard | 34 | 
| 345 | san_dorian_marriage_certificate | 34 | 
| 346 | bastokan_marriage_certificate | 34 | 
| 347 | windurstian_marriage_certificate | 34 | 
| 356 | set_of_guild_flyers | 34 | 
| 365 | poele_classique | 34 | 
| 366 | kanonenofen | 34 | 
| 367 | pot_topper | 34 | 
| 375 | cs_gold_stand | 34 | 
| 376 | cs_mythril_stand | 34 | 
| 377 | cs_bronze_stand | 34 | 
| 415 | aldebaran_horn | 34 | 
| 426 | orchestrion | 34 | 
| 438 | orblight | 34 | 
| 441 | taru_tot_toyset | 34 | 
| 451 | egg_stool | 34 | 
| 452 | egg_table | 34 | 
| 453 | egg_locker | 34 | 
| 454 | egg_lantern | 34 | 
| 455 | egg_buffet | 34 | 
| 456 | pepo_lantern | 34 | 
| 457 | cushaw_lantern | 34 | 
| 458 | calabazilla_lantern | 34 | 
| 461 | buffalo_milk_case | 34 | 
| 10593 | decennial_tights | 37 | 
| 21346 | leafkin_grenade | 15 |

File: sql/item_weapon.sql (hunk 1 above)
Comment (Ninjistix) [reply sql/item_weapon.sql]: will fix but i didnt set any ilv stats, whatever i used to update this list didn't include those

**Comment by Ninjistix** [comment]
@TeoTwawki 
img_20180410_005745
img_20180410_005758

**Comment by zynjec** [comment]
It's a mistranslation in English, it's `ポポトフライ` (popoto fry) in Japanese, and popoto in every other instance in the game, so it's more accurate to leave it as-is despite the long name being pototo.

File: sql/item_weapon.sql [hunk 30]
```diff
@@ -52,7 +52,7 @@ INSERT INTO `item_weapon` VALUES (16387,'poison_cesti',1,0,0,0,0,4,1,528,6,0);
 INSERT INTO `item_weapon` VALUES (16388,'himantes',1,0,0,0,0,4,1,528,7,0);
 INSERT INTO `item_weapon` VALUES (16389,'coeurl_cesti',1,0,0,0,0,4,1,528,8,0);
 INSERT INTO `item_weapon` VALUES (16390,'bronze_knuckles',1,0,0,0,0,4,1,576,5,0);
-INSERT INTO `item_weapon` VALUES (16391,'brass_knuckles',1,0,0,0,0,4,1,576,7,0);
```
Comment (TeoTwawki) [inline sql/item_weapon.sql]: :confounded: 
both bronze and brass NQ's have same delay of `+96` the plus indicating it is added to the h2h base delay.

Comment (TeoTwawki) [reply sql/item_weapon.sql]: this pr is just to darn big to even review all at once.

**Comment by Ninjistix** [comment]
its fine someone else can take a look at it, i no longer have the time to update this.

**Pull request closed without merging**